core/src/main/java/nl/nn/adapterframework/util/MessageDataSource.java

```diff
@@ -15,20 +15,20 @@
 */
 package nl.nn.adapterframework.util;
 
+import java.io.Closeable;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
 
-import javax.activation.DataSource;
-
+import org.apache.commons.lang3.NotImplementedException;
```

Comment:
Perhaps `java.lang.UnsupportedOperationException` is more applicable than an Apache Commons specific exception?